multiqc/utils/search_patterns.yaml

```diff
@@ -730,6 +730,10 @@ sargasso:
 sentieon/alignment_metrics:
   contents: "--algo AlignmentStat"
   shared: true
+sentieon/dedup:
+  contents: "--algo Dedup"
+  num_lines: 3
+  shared: true
```

Comment:
If these logs can be shared, then setting `num_lines` doesn't really make sense - if the output was appended half way down an existing file then it won't be in the first 3 lines.

If Sentieon always creates these files then remove `shared` and keep `num_lines` (maybe also for the other blocks above and below?). If a likely use case is that the output from multiple samples / tools could be appended to a single file, remove `num_lines`

Reply:
Thanks for you prompt reply, @ewels. I probably didn't (don't) understand the purpose of `shared` and `num_lines` completely 😬 The new Sentieon-Dedup MultiQC-module only uses the first three lines from the sentieon-dedup-metrics files, that is why I set `num_lines` to three.  

Concerning the option `shared`, I think I looked at this:

https://github.com/ewels/MultiQC/blob/7129bd10420149c032e380bef57f9c9ca0044d33/docs/core/development/modules.md?plain=1#L381-L382

and I figured if some other MultiQC-module wanted to use the same log-file, then fine. Moreover, the other Sentieon-MultiQC-modules had `shared` set to `true` so I also just did that.

On re-reading your explanation and the docs, I get the impression that `shared` is actually more a way of having MultiQC handle the case where several pipeline-tools write to the same log/metrics/summary-file. If that is indeed the case, then I guess `shared` should be removed from all the Sentieon MultiQC-modules in the `search_patterns.yaml` 🤔

Reply:
@ewels : So I should just the field `num_lines` and `shared` for `sentieon`?